contracts/storage/position.ral

```diff
@@ -14,7 +14,8 @@ struct PositionState {
 Contract Position(
     admin: Address,
     mut position: PositionState,
-    mut isActive: Bool
+    mut isActive: Bool,
+    clammContractInstance: CLAMM
```

Comment:
you can name it simply `clamm`

Reply:
There is already a global variable named `clamm` that's why it needs to be diffrent